cypress/integration/Qwiki /risk.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { contains } from "cypress/types/jquery";
+import { Qwiki } from "../../tester/Qwiki";
+
+it("can create a page", () => {
+  const qwiki = new Qwiki("https://dev.testing.modac.eu/");
+  const overview = qwiki.login({username: "admin",password: "modell-aachen"});
+  const risk = overview.createRisk({ title: "Risk123" });
+  risk.isARisk({ title: "Risk123" });
+});
```

Comment:
```suggestion
it("can create a page", () => {
  const qwiki = new Qwiki("https://dev.testing.modac.eu/");
  const overview = qwiki.login({username: "admin",password: "modell-aachen"});
  const risk = overview.submitPossibleRisk({ title: "Risk123", linkedProcess: ..... });
  risk.hasInformations({
   title: "Risk123",
   linkedProcess: '....',
   reportedBy: '....',
    });
    risk.isAPossibleRisk();
});
```

Am Beispiel von _Ziele und Strategien_